src/Mod/Path/PathScripts/PathDressupZCorrect.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, obj):
         obj.addProperty("App::PropertyFile", "probefile", "ProbeData", QtCore.QT_TRANSLATE_NOOP("Path_DressupZCorrect", "The point file from the surface probing."))
         obj.Proxy = self
         obj.addProperty("Part::PropertyPartShape", "interpSurface", "Path")
-        obj.setEditorMode('interpSurface', 2)  # hide
+        #obj.setEditorMode('interpSurface', 2)  # hide
```

Comment:
By the way, to show and hide properties I suggest using the new method that realthunder introduced; it's more readable than the numeric editor "modes".
```py
obj.setPropertyStatus("interpSurface", "Hidden")
obj.setPropertyStatus("interpSurface", "-Hidden")  # Remove hidden, that is, show
```

`Hidden` and `ReadOnly` are the most useful ones; I think `Immutable` is a completely fixed one; it can't be changed from the Python console either.
```py
obj.getPropertyStatus()
['Hidden', 'Immutable', 'LockDynamic', 'MaterialEdit', 'NoMaterialListEdit', 'NoModify', 'Output', 'PartialTrigger', 'ReadOnly', 'Transient']
```